Description: Fix a few typos on the manpages and source code.
Author: Giovani Augusto Ferreira <giovani@debian.org>
Last-Update: 2016-07-07
Index: dislocker-0.7.1/man/linux/dislocker-file.1
===================================================================
--- dislocker-0.7.1.orig/man/linux/dislocker-file.1
+++ dislocker-0.7.1/man/linux/dislocker-file.1
@@ -2,7 +2,7 @@
 .\"
 .TH DISLOCKER 1 2011-09-07 "Linux" "DISLOCKER"
 .SH NAME
-Dislocker file - Read BitLocker encrypted volumes under Linux, OSX and FreeBSD.
+Dislocker-file \- Read BitLocker encrypted volumes under Linux, OSX and FreeBSD.
 .SH SYNOPSIS
 dislocker-file [-hqrsv] [-l \fILOG_FILE\fR] [-O \fIOFFSET\fR] [-V \fIVOLUME\fR \fIDECRYPTMETHOD\fR -F[\fIN\fR]] [--] \fINTFS_FILE\fR
 
@@ -16,7 +16,7 @@ This NTFS file won't have any link with
 .SH OPTIONS
 For program's options description, see dislocker-fuse(1). The only change in the command line is the last argument, which in this case is the \fINTFS_FILE\fR argument:
 .PP
-.TB
+.TP
 .B NTFS_FILE
 the newly created file where NTFS data will be put to, once decrypted from the BitLocker encrypted volume.
 .SH EXAMPLES
Index: dislocker-0.7.1/man/linux/dislocker-fuse.1
===================================================================
--- dislocker-0.7.1.orig/man/linux/dislocker-fuse.1
+++ dislocker-0.7.1/man/linux/dislocker-fuse.1
@@ -2,7 +2,7 @@
 .\"
 .TH DISLOCKER-FUSE 1 2011-09-07 "Linux" "DISLOCKER-FUSE"
 .SH NAME
-Dislocker fuse - Read/write BitLocker encrypted volumes under Linux, OSX and FreeBSD.
+Dislocker-fuse \- Read/write BitLocker encrypted volumes under Linux, OSX and FreeBSD.
 .SH SYNOPSIS
 dislocker-fuse [-hqrsv] [-l \fILOG_FILE\fR] [-O \fIOFFSET\fR] [-V \fIVOLUME\fR \fIDECRYPTMETHOD\fR -F[\fIN\fR]] [-- \fIARGS\fR...]
 
@@ -49,7 +49,7 @@ do NOT display any information.
 This option has priority on any previous `\fB-v\fR'. One probably wants to check the return value of the program when using this option
 .TP
 .B -r, --readonly
-do not allow to write on the BitLocker volume (read only mode)
+do not allow one to write on the BitLocker volume (read only mode)
 .TP
 .B -s, --stateok
 do not check the volume's state, assume it's ok to mount it.
Index: dislocker-0.7.1/src/metadata/datums.c
===================================================================
--- dislocker-0.7.1.orig/src/metadata/datums.c
+++ dislocker-0.7.1/src/metadata/datums.c
@@ -326,7 +326,7 @@ void print_datum_key(DIS_LOGS level, voi
 	datum_key_t* datum = (datum_key_t*) vdatum;
 	char* cipher_str = cipherstr((cipher_t) datum->algo);
 
-	dis_printf(level, "Unkown: \n");
+	dis_printf(level, "Unknown: \n");
 	hexdump(level, (void*) &datum->padd, 2);
 	dis_printf(level, "Algo: %s (%#hx)\n", cipher_str, datum->algo);
 	dis_printf(level, "Key:\n");
@@ -367,7 +367,7 @@ void print_datum_stretch_key(DIS_LOGS le
 {
 	datum_stretch_key_t* datum = (datum_stretch_key_t*) vdatum;
 
-	dis_printf(level, "Unkown: \n");
+	dis_printf(level, "Unknown: \n");
 	hexdump(level, (void*) &datum->padd, 2);
 	dis_printf(level, "Algo: %#x\n", datum->algo);
 	dis_printf(level, "Salt: \n");
Index: dislocker-0.7.1/src/metadata/metadata.c
===================================================================
--- dislocker-0.7.1.orig/src/metadata/metadata.c
+++ dislocker-0.7.1/src/metadata/metadata.c
@@ -202,7 +202,7 @@ int dis_metadata_initialize(dis_metadata
 	{
 		dis_printf(
 			L_CRITICAL,
-			"A problem occured during the retrieving of metadata. Abort.\n"
+			"A problem occurred during the retrieving of metadata. Abort.\n"
 		);
 		return DIS_RET_ERROR_METADATA_CHECK;
 	}
Index: dislocker-0.7.1/src/metadata/print_metadata.c
===================================================================
--- dislocker-0.7.1.orig/src/metadata/print_metadata.c
+++ dislocker-0.7.1/src/metadata/print_metadata.c
@@ -59,7 +59,7 @@ void print_volume_header(DIS_LOGS level,
 	format_guid(volume_header->guid, rec_id);
 
 
-	dis_printf(level, "=====[ Volume header informations ]=====\n");
+	dis_printf(level, "=====[ Volume header information ]=====\n");
 	dis_printf(level, "  Signature: '%.8s'\n", volume_header->signature);
 	dis_printf(level, "  Sector size: 0x%1$04x (%1$hu) bytes\n", volume_header->sector_size);
 	dis_printf(level, "  Sector per cluster: 0x%1$02x (%1$hhu) bytes\n", volume_header->sectors_per_cluster);
@@ -122,7 +122,7 @@ void print_information(DIS_LOGS level, d
 	dis_printf(level, "  Current state: %s (%hu)\n", get_state_str(information->curr_state), information->curr_state);
 	dis_printf(level, "  Next state: %s (%hu)\n",    get_state_str(information->next_state), information->next_state);
 	dis_printf(level, "  Encrypted volume size: %1$" PRIu64 " bytes (%1$#" PRIx64 "), ~%2$" PRIu64 " MB\n", information->encrypted_volume_size, information->encrypted_volume_size / (1024*1024));
-	dis_printf(level, "  Size of convertion region: %1$#x (%1$u)\n", information->convert_size);
+	dis_printf(level, "  Size of conversion region: %1$#x (%1$u)\n", information->convert_size);
 	dis_printf(level, "  Number of boot sectors backuped: %1$u sectors (%1$#x)\n", information->nb_backup_sectors);
 	dis_printf(level, "  First metadata header offset:  %#" PRIx64 "\n", information->information_off[0]);
 	dis_printf(level, "  Second metadata header offset: %#" PRIx64 "\n", information->information_off[1]);
@@ -187,7 +187,7 @@ void print_eow_infos(DIS_LOGS level, dis
 
 	bitlocker_eow_infos_t* eow_infos = dis_meta->eow_information;
 
-	dis_printf(level, "=======================[ BitLocker EOW informations ]========================\n");
+	dis_printf(level, "=======================[ BitLocker EOW information ]========================\n");
 	dis_printf(level, "  Signature: '%.8s'\n", eow_infos->signature);
 	dis_printf(level, "  Structure size: 0x%1$04x (%1$hu)\n", eow_infos->header_size);
 	dis_printf(level, "  On-disk size: 0x%1$04x (%1$hu)\n", eow_infos->infos_size);
@@ -243,7 +243,7 @@ void print_data(DIS_LOGS level, dis_meta
 			break;
 
 		dis_printf(level, "\n");
-		dis_printf(level, "=======[ Datum n°%d informations ]=======\n", ++loop);
+		dis_printf(level, "=======[ Datum n°%d information ]=======\n", ++loop);
 		print_one_datum(level, data);
 		dis_printf(level, "=========================================\n");
 
Index: dislocker-0.7.1/src/config.c
===================================================================
--- dislocker-0.7.1.orig/src/config.c
+++ dislocker-0.7.1/src/config.c
@@ -181,7 +181,7 @@ PROGNAME " by " AUTHOR ", v" VERSION " (
 "    -p, --recovery-password=[RECOVERY_PASSWORD]\n"
 "                          decrypt volume using the recovery password method\n"
 "    -q, --quiet           do NOT display anything\n"
-"    -r, --readonly        do not allow to write on the BitLocker volume\n"
+"    -r, --readonly        do not allow one to write on the BitLocker volume\n"
 "    -s, --stateok         do not check the volume's state, assume it's ok to mount it\n"
 "    -u, --user-password=[USER_PASSWORD]\n"
 "                          decrypt volume using the user password method\n"
Index: dislocker-0.7.1/man/linux/dislocker-find.1
===================================================================
--- dislocker-0.7.1.orig/man/linux/dislocker-find.1
+++ dislocker-0.7.1/man/linux/dislocker-find.1
@@ -2,7 +2,7 @@
 .\"
 .TH DISLOCKER-FIND 1 2011-09-07 "Linux" "DISLOCKER-FIND"
 .SH NAME
-Dislocker find - Find BitLocker-encrypted volumes.
+Dislocker-find \- Find BitLocker-encrypted volumes.
 .SH SYNOPSIS
 dislocker-find [-h] [files...]
 .SH DESCRIPTION
